routes/api/articles.js

```diff
@@ -24,6 +24,7 @@ router.get(
   verifyJwt({ tokenRequired: false }),
   ArticleController.searchArticles
 );
+router.get('/report', verifyJwt({ tokenRequired: true }), ArticleController.getArticleReports);
```

Comment:
@filschristian Since you are not using try and catch in your controller, add asyncHandler to your router.

Reply:
Good. Thanks. I will add it